test/parallel/test-child-process-execFile-promisified-abortController.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const { promisify } = require('util');
+const execFile = require('child_process').execFile;
+const fixtures = require('../common/fixtures');
+
+const echoFixture = fixtures.path('echo.js');
+const promisified = promisify(execFile);
+const invalidArgTypeError = {
+  code: 'ERR_INVALID_ARG_TYPE',
+  name: 'TypeError'
+};
+
+{
+  // Verify that the signal option works properly
+  const ac = new AbortController();
+  const signal = ac.signal;
+  const promise = promisified(process.execPath, [echoFixture, 0], { signal });
+
+  ac.abort();
+
+  promise.catch(common.mustCall((e) => {
+    assert.strictEqual(e.name, 'AbortError');
+  }));
+}
+
+{
+  // Verify that the signal option works properly when already aborted
+  const ac = new AbortController();
+  const { signal } = ac;
+  ac.abort();
+
+  const promise = promisified(process.execPath, [echoFixture, 0], { signal });
+
+  promise.catch(common.mustCall((e) => {
+    assert.strictEqual(e.name, 'AbortError');
+  }));
```

Comment:
Use `assert.rejects` for this please (and the other tests involving rejecting promises).
```suggestion
  assert.rejects(
    () => promisified(process.execPath, [echoFixture, 0], { signal },
    { name: 'AbortError' }
  ).then(common.mustCall());
```

Reply:
Done in 72ff68a
Please have a look 🙂